### PR TITLE
Add gevent to python.yaml

### PR DESCRIPTION
**Pull request by manacoa**
gevent is a coroutine based library for multitasking Python.

http://www.gevent.org/

### CODE REVIEW

**Comment by tfoote** [comment]
It looks like gevent is available via debian package. By default we prefer to use the native package manager's version. 

Ubuntu: http://packages.ubuntu.com/trusty/python-gevent
It's also in Jessie https://packages.debian.org/jessie/python-gevent
And Fedora: https://admin.fedoraproject.org/pkgdb/package/rpms/python-gevent/

I've seen several sources that suggest osx needs a combo of `libevent` from homebrew and `gevent` from pip. Which likely suggests that the generic pip rule on any platform needs libevent as a dependency since that can't be tracked in the pip exclusive dependencies.

**Comment by manacoa** [comment]
Hi Tully,

I've changed the pip installs to apt-get. Looking good?

Simon

**Comment by tfoote** [comment]
Thanks for switching to the apt based packages. Since we're using the default package manager it's better to use the compact representation `codename: [package]` for each. It's more compact which makes it more readable and maintainable.

File: rosdep/python.yaml [hunk 1]
```diff
@@ -663,6 +663,16 @@ python-gdown:
   ubuntu:
     pip:
       packages: [gdown]
+python-gevent:
+  fedora:
+    apt:
```
Comment (tfoote) [inline rosdep/python.yaml]: apt is not a valid installer on fedora. If you switch to the compact format in my other comment this problem will disappear.

**Comment by manacoa** [comment]
How does this look?

**Comment by tfoote** [comment]
The compact format looks good. Sorry I missed in the last round, but the key for debian should be `debian`  not `jessie`. (Note it will need to jump up to stay in alphabetical order too.)

**Comment by manacoa** [comment]
We're getting there!

**Comment by tfoote** [comment]
+1

Thanks for iterating!

**Comment by SimonBirrell** [comment]
Thank you, sir!

**Pull request merged**